volatility/framework/plugins/mac/psaux.py

```diff
@@ -80,8 +80,11 @@ def _generator(self, tasks: Iterator[Any]) -> Generator[Tuple[int, Tuple[int, st
                     args.append(arg)
 
                 argc = argc - 1
-
-            args_str = " ".join([s.decode("utf-8") for s in args])
+                
+            try:
+                args_str = " ".join([s.decode("utf-8") for s in args])
+            except:
```

Comment:
This `except` is too general, and should really be `except UnicodeDecodeError` if that's the only exception you're trying to catch.

Reply:
I believe this needs to get refactored even further as it will be bail out on processing/returning all arguments, even if only one cannot be decoded.